pommerman/graphics.py

```diff
@@ -372,9 +372,19 @@ def render_dead_alive(self):
         dead.width = image_size
         dead.height = image_size
         sprites = []
-
-        for agent in self._agents:
-            x = self.board_right(x_offset=-3) - (4 - agent.agent_id) * (
+        
+        if self._game_type is constants.GameType.FFA:
+            agents = self._agents
+        else:
+            agents = [
+                self._agents[0],
+                self._agents[2],
+                self._agents[1],
+                self._agents[3]
+            ]
+
+        for index, agent in enumerate(agents):
+            x = self.board_right(x_offset=-3) - (4 - index) * (
```

Comment:
A comment right here saying why you are doing this funky computation would be helpful.